src/v3/src/transformer/oktaVerify/transformOktaVerifyChannelSelection.ts

```diff
@@ -32,17 +29,16 @@ const CHANNEL_TO_LABEL_KEY_MAP: { [channel: string]: string } = {
   sms: 'oie.enroll.okta_verify.select.channel.sms.label',
 };
 
-export const transformOktaVerifyChannelSelection: IdxStepTransformer = (transaction) => {
-  const { context } = transaction;
-  // TODO: OKTA-503490 temporary sln to access missing relatesTo obj
-  const authenticator = context?.currentAuthenticator?.value;
-  const selectedChannel = authenticator?.contextualData?.selectedChannel;
-  const isAndroidOrIOSView = isAndroidOrIOS();
+export const transformOktaVerifyChannelSelection: IdxStepTransformer = ({
+  prevTransaction,
+  formBag,
+}) => {
+  const { data, uischema } = formBag;
 
-  const channelSelectionStep = transaction.availableSteps?.find(
-    ({ name }) => name === IDX_STEP.SELECT_ENROLLMENT_CHANNEL,
-  );
-  const formBag = transformStepInputs(channelSelectionStep);
+  const isAndroidOrIOSView = isAndroidOrIOS();
+  // eslint-disable-next-line max-len
+  const selectedChannel = prevTransaction?.context.currentAuthenticator.value.contextualData?.selectedChannel as string;
```

Comment:
Can this be renamed to `previouslySelectedChannel`? 

If I understand this correctly, this is the last selected channel, not the currently selected channel?